### PR TITLE
Add ability to provide custom templates

### PR DESCRIPTION
**Pull request by cakeinpanic**
Add ability to provide custom templates

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -14,6 +14,6 @@
 	],
     "dependencies": {
 	  "dxjs": "devex-web-frontend/dxjs#^2.0.3",
-	  "dropdown": "devex-web-frontend/dropdown#^2.4.1"
+	  "dropdown": "devex-web-frontend/dropdown#svg"
```
Comment (raveclassic) [inline bower.json]: don't forget to remove this dependency

**Comment by raveclassic** [comment]
`npm test` fails

File: src/js/colorpicker.js [hunk 2]
```diff
@@ -57,14 +65,16 @@ var Colorpicker = (function(DX, window, document, undefined) {
 	 * @constructor Colorpicker
 	 * @param {HTMLInputElement} input
 	 * @param {Array} colorList
+	 * @param {Object} customTemplateConfig
 	 */
-	return function Colorpicker(input, colorList) {
```
Comment (raveclassic) [inline src/js/colorpicker.js:61]: fix jsdoc and update documentation

Comment (cakeinpanic) [inline src/js/colorpicker.js:61]: have been done already, look again)

**Pull request merged**